plugins/s3-viewer-backend/src/service/router.ts

```diff
@@ -81,10 +79,6 @@ export async function createPluginPermissions({
   config,
 }: RouterPermissionOptions): Promise<express.Router> {
   const discovery = HostDiscovery.fromConfig(config);
-  return await createPermissionPlugin({
-    config: config,
-    logger: loggerToWinstonLogger(logger),
-    discovery: discovery,
-    policy: new TestPermissionPolicy(),
-  });
+  const policy = new TestPermissionPolicy();
```

Comment:
This is only used in the `standaloneServer.ts` file for the local development setup.

However, let me give that a second check. Looks like it's broken because the guest provider is missing

Reply:
Well... Not a problem that seems to be solvable at the moment :confused: 

I've been trying a bunch of upstream plugins which also have the `dev` setup, and all of them have the exact same problem: The authentication page pops up asking you to login as guest, and then a 500 error. So looks like this is a rather common problem for all.

Reply:
> This is only used in the standaloneServer.ts file for the local development setup.

Ok, looks fine then. :)

> I've been trying a bunch of upstream plugins which also have the dev setup, and all of them have the exact same problem: The authentication page pops up asking you to login as guest, and then a 500 error. So looks like this is a rather common problem for all.

That sounds rather odd, could you ask in the Discord or open a ticket, breaking dev mode everywhere seems unintended. 😅

Reply:
I think so too. I will merge this for now, and try to find a way to solve the issue, maybe there's something I missed during other releases, but I'm not aware of anything like that